components/feature/addons/src/main/res/values/strings.xml

```diff
@@ -113,6 +113,10 @@
     <string name="mozac_feature_addons_updater_notification_content" tools:ignore="PluralsCandidate">%1$d new permissions are required</string>
     <!-- Name of the "notification channel" used for displaying a notification for updating an add-on. See https://developer.android.com/training/notify-user/channels -->
     <string name="mozac_feature_addons_updater_notification_channel">Add-on updates</string>
+    <!-- Name of the "notification channel" used for displaying a notification for new supported add-on. See https://developer.android.com/training/notify-user/channels -->
+    <string name="mozac_feature_addons_supported_checker_notification_channel">Supported add-ons checker</string>
+    <!-- The tile of the notification, this will be shown to the user when one more new supported add-ons are available.-->
+    <string name="mozac_feature_addons_supported_checker_notification_title">One or some of your add-ons are now available</string>
```

Comment:
nit: Hm did we get this String from UX? It reads a bit strange to me e.g. just `Some of your add-ons...` or `One or more of your ...`. But ignore if that's the String we should use.

Reply:
I took them from vesta's comment, I guess they should be hard-coded, right?  https://github.com/mozilla-mobile/android-components/issues/5434#issuecomment-590469047